### ISSUE TITLE
Keyboard navigation (accessibility)

### ISSUE THREAD

**Issue by petterill** [doczjs/docz#1512]
Feature request
---

The menu of `gatsby-theme-docz` is problematic with regards to web accessibility (a11y) as it cannot be fully navigated with a keyboard. 

Try using tab to select the next element. All top level menu items that can be expanded/collapsed and have child items will be overlooked with keyboard navigation. A top level menu item cannot be expanded to select the child items. Once an expandable menu item is expanded by clicking with a mouse, the child items can be accessed by keyboard, but the top level item cannot. 

Menu items that do not have child items can be selected. The headline, search box and light/dark toggle can be selected and work as expected.

This lack of keyboard navgation is especially problematic for all public sector websites in the EU that are required to be accessible by the standards of WCAG 2.0. We at the City of Helsinki, Finland have started using docz for a couple of our documentation sites but this lack of accessibility is becoming an issue.
